servers/execution/runner/script/start.bash

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+printf "starting the node application\n\n"
+
+#npx cross-env NODE_OPTIONS="--es-module-specifier-resolution=node  --experimental-modules --experimental-specifier-resolution=node" NODE_NO_WARNINGS=1 node dist/src/main.js
+
+MOD_RES="--es-module-specifier-resolution=node"
+EXP="--experimental-modules"
+EXP_RES="--experimental-specifier-resolution=node"
+npx cross-env \
+  NODE_OPTIONS="$MOD_RES $EXP $EXP_RES" \
+  NODE_NO_WARNINGS=1 \
+  node dist/src/main.js
```

Comment:
Should it run 'main' or 'runner'?

Reply:
It should run `main.js`. I will change this and check again.